style.css

```diff
@@ -0,0 +1,832 @@
+* {
+  padding: 0;
+  margin: 0;
+  font-family: 'Lato', sans-serif;
+}
+
+.navbar {
+  display: none;
+}
+
+.navbar1 {
+  width: 100vw;
+  height: 30px;
+  background-color: #272a31;
```

Comment:
- [ ] You have 6 instances of this rule `background-color: #272a31;` ,  Please write DRY code. You can achieve this by writing a background-color class and assigning it to all your elements that need to have the same background. 🔍 Dry code is easy to maintain and results in less code 💯 😃

Reply:
Hello Francis,

Done.  Thanks